cmd/gmnhg/main.go

```diff
@@ -150,6 +162,15 @@ func writeFile(dst string, contents []byte) error {
 	return nil
 }
 
+func hasSubPath(paths []string, path string) bool {
```

Comment:
Does your editor run `gofmt` on save? `hasSubPath` appears to be indented with spaces, whilst the rest of the code is done with tabs (as most Go code does).

Reply:
Thanks, I need to invest some time into my setup. I thought this was going to be a small change, a few lines at most, so I didn't even set up a proper environment... but hey, now I'm learning go, so that's a win. :)